site/docs/vendors.md

```diff
@@ -61,6 +61,10 @@ the same copy of data using Spark and run analytics or AI with our
 [Machine Learning](https://www.cloudera.com/products/machine-learning.html) tools on private
 or any public cloud.
 
+### [Confluent](https://confluent.io)
+
+[Confluent](https://www.confluent.io/) provides [Tableflow](https://www.confluent.io/product/tableflow/), a managed service for streaming data from Apache Kafka topics into Apache Iceberg tables. Tableflow automates schema evolution through Schema Registry integration and handles table maintenance tasks including compaction automatically
```

Comment:
just a style nit so we're consistent with the other sections but could we make sure the last sentence ends in a period.

Reply:
thanks, done.